CONTRIBUTING_TO_OPENSHIFT_PLUGINS.md

```diff
@@ -0,0 +1,280 @@
+# Development and release process for the OpenShift Jenkins plugins
+
+The OpenShift organization currently maintains for plugins related to integration between V3 OpenShift and V2 Jenkins:
+
+* [OpenShift Client Plugin](https://github.com/openshift/jenkins-client-plugin)
+* [OpenShift Sync Plugin](https://github.com/openshift/jenkins-sync-plugin)
+* [OpenShift Login Plugin](https://github.com/openshift/jenkins-openshift-login-plugin)
+* [OpenShift Pipeline Plugin (only in maintenance at this point)](https://github.com/openshift/jenkins-plugin)
```

Comment:
move (only in maintenance at this point) after the link and maybe put in italics?